c/tree.h

```diff
@@ -1,6 +1,10 @@
 #include "EXTERN.h"
-#include "XSUB.h"
+// clang-format wants to change the order of these headers. However order is
+// important.
+// clang-format off
 #include "perl.h"
+#include "XSUB.h"
```

Comment:
What do you say to something like this instead? https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/w0b63ZHuxgo

I actually don't mind this and think it's pretty clear, but a comment + newline before each header would work too.

Reply:
Yeah, that's cleaner. I changed it!